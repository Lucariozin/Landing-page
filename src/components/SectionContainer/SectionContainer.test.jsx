import { render } from '@testing-library/react';
import { SectionContainer } from '.';

describe('<SectionContainer />', () => {
  it('should render <SectionContainer /> with background white', () => {
    const { container } = render(<SectionContainer>Hello</SectionContainer>);

    expect(container.firstChild).toHaveStyleRule(
      'background-color',
      'var(--white)',
    );
  });

  it('should render <SectionContainer /> with background dark', () => {
    const { container } = render(
      <SectionContainer backgroundDark>Hello</SectionContainer>,
    );

    expect(container.firstChild).toHaveStyle({
      'background-image': 'url(assets/img/background.svg)',
      'background-size': 'cover',
      'background-position': 'center center',
    });
  });

  it('should to match snapshot', () => {
    const { container } = render(<SectionContainer>Hello</SectionContainer>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
