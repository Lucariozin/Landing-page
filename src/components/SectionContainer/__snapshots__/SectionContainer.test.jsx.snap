// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionContainer /> should to match snapshot 1`] = `
.c0 {
  width: 100%;
  min-height: 100vh;
  padding: 1rem;
  padding-top: 8rem;
  background-color: var(--white);
  color: var(--primary-color);
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
}

<div
  class="c0"
  id=""
>
  <div
    class="c1"
  >
    Hello
  </div>
</div>
`;
